docs/products/clickhouse/howto/configure-backup.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Configure Aiven for ClickHouse® backup scheduling settings
```

Comment:
Could we update the title to something like "Configure backup scheduling in Aiven for ClickHouse®" or "Configure Aiven for ClickHouse® backup schedule"? I feel that "scheduling settings" is a bit redundant, since scheduling itself implies configuration or settings